lib/__tests__/client.unit.test.js

```diff
@@ -395,6 +395,30 @@ testSimpleReq({
   requestOptions: { method: 'post', params: { malicious: true } }
 })
 
+testSimpleReq({
+  name: 'addTagToAlert',
+  id: 'abc123',
+  args: ['abc123', 'test tag'],
+  url: '/1/alert/abc123/tags/test%20tag',
+  requestOptions: { method: 'post' }
+})
+
+testSimpleReq({
+  name: 'removeTagFromAlert',
+  id: 'abc123',
+  args: ['abc123', 'test tag'],
+  url: '/1/alert/abc123/tag/test%20tag',
+  requestOptions: { method: 'delete' }
+})
+
+testSimpleReq({
+  name: 'addNotesToAlert',
+  id: 'abc123',
+  args: ['abc123', 'note'],
+  url: '/1/alert/abc123/notes',
+  requestOptions: { method: 'put', params: { notes: 'note' } }
```

Comment:
sorry it's taking me so long to get around to review! but i think this test should include `body` in the requestOptions